### ISSUE TITLE
Unhandled command:  PROXY IPV4 TCP 10.0.0.1 53

### ISSUE THREAD

**Issue by batkedev** [paradust7/minetest-wasm-sample-proxy#1]
If I open the wasm client locally and try to connect to a self hosted minetest server, the proxy connection doesn't work:

```sh
Proxy listening on port 8888
[CLIENT 1] New client from  [ '', '::ffff:127.0.0.1' ]
[CLIENT 1] Unhandled command:  PROXY IPV4 TCP 10.0.0.1 53
```

Here the wasm client output of the browser console of https://localhost/index.html (serves the www build folder):

```sh
Uncaught 50077280 [minetest.js:6:5](https://localhost/fca7c8c9d007/minetest.js)
    quit_ https://localhost/minetest/fca7c8c9d007/minetest.js:6
    handleException https://localhost/fca7c8c9d007/minetest.js:1255
    callUserCallback https://localhost/fca7c8c9d007/minetest.js:2240
    ...
```
`this.proxyUrl` in launcher.js is set to `ws://localhost:8888`. The original url `wss://minetest.dustlabs.io/proxy` works.

https://github.com/paradust7/minetest-wasm/blob/e8f754e1c59517be978935935849be61352cfc40/static/launcher.js#L576

`this.vpn` is `null` - could that be the reason?

https://github.com/paradust7/minetest-wasm/blob/e8f754e1c59517be978935935849be61352cfc40/static/launcher.js#L573

Are some additional settings necessary?

**Comment by paradust7** [comment]
Just to make sure I understand, you're running a minetest server outside of the browser? On localhost?

**Comment by batkedev** [comment]
> Just to make sure I understand, you're running a minetest server outside of the browser? On localhost?

Correct, I run a minetest server outside of the browser, accessible via a public domain (not on local host). The client for the browser and the proxy run local.

As I mentioned, with one of your public proxy servers everything works fine, without any other changes.
